### DISCUSSION TITLE
Remix landing page under the hood?

### DISCUSSION THREAD

**Discussion by nik72619c** [remix-run/remix#3714] [Q&A] [answered]
Hi 👋🏼 
I really loved the concept of Remix and also, the UX of the [website](https://remix.run/). Would love to know more about it's internals or how these animations are working.
Would also love to contribute to them.
Thanks!

**Comment by machour** [comment, marked as answer]
The website is not open sourced, as stated in #3663

**Reply by stereokai** [reply]
Will you at least kindly share the library that was used for orchestrating the scrolling animations? Thanks 😁

**Reply by kiliman** [reply]
I believe @ryanflorence wrote that custom code himself.

**Reply by stereokai** [reply]
Very nice work then :)
